x/ibc-transfer/keeper/relay.go

```diff
@@ -138,8 +143,8 @@ func (k Keeper) createOutgoingPacket(
 		sourceChannel,
 		destinationPort,
 		destinationChannel,
-		destHeight+DefaultPacketTimeoutHeight,
-		DefaultPacketTimeoutTimestamp,
+		destHeight+timeoutHeight,
```

Comment:
not sure if this addition still applies? @cwgoes

Reply:
Let's get rid of it, no longer necessary if the timeout is user-specified